lxd/images.go

```diff
@@ -2131,7 +2131,6 @@ func pruneExpiredImages(ctx context.Context, d *Daemon, op *operations.Operation
 }
 
 func pruneExpiredImagesInProject(ctx context.Context, d *Daemon, project db.Project, config map[string]string, op *operations.Operation) error {
```

Comment:
@masnax this looks like it can be sent up as a standalone PR, if possible can you do this so we can get this PR merged in smaller chunks. Thanks